source/docs/software/driverstation/driver-station-errors-warnings.rst

```diff
@@ -50,11 +50,11 @@ This message is printed each time the DS begins communicating with the roboRIO.
 
   WARNING<Code>44008 occurred at FRC_NetworkCommunications
   <radioLostEvents> 19.004<radioSeenEvents> 0.000
-  FRC: Robot radio dectection times
+  FRC: Robot radio detection times
```

Comment:
This is the output from the DS and shouldn't be corrected unless the DS has changed